RecordingApp/app/src/main/java/wycliffeassociates/recordingapp/project/ProjectWizardActivity.java

```diff
@@ -119,8 +121,20 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
         }
     }
 
+    private void closeKeyboardAndSearch(){
```

Comment:
Since the function is doing two separate things, I think it's wise to split it into `closeKeyboard()` and `resetSearchText()`. Or something like that.

Also, should there be a space before the opening curly bracket?